apps/clockbyhenry/clock_by_henry.star

```diff
@@ -0,0 +1,499 @@
+    season = int(now.format("1-02").replace("-", ""))
+    if season >= 320 and season < 621:
+        season = 1
+    elif season >= 621 and season < 923:
+        season = 2
+    elif season >= 923 and season < 1221:
+        season = 3
+    else:
+        season = 0
+
+    phase = int(now.format("15"))
+    if phase >= 6 and phase < 18:
+        phase = "d"
+    else:
+        phase = "n"
+
+    # get the base color set
+    season_theme = SEASON_THEMES.get(
+        config.get(P_SEASON_THEME),
+        SEASON_THEMES[DEFAULT_SEASON_THEME],
+    )
+
+    time_color = color_of(config.get(phase), DEFAULT_TIME_COLOR[phase])
+    date_color = color_of(
+        config.get("s%d%s" % (season, phase)),
+        season_theme[phase][season],
+    )
+
+    # generate the widget for the app
+    return render.Root(
+        delay = 1000,
+        child = render.Column(
+            main_align = "center",
+            expanded = True,
+            children = [
+                render.Row(
+                    main_align = "center",
+                    expanded = True,
+                    children = [
+                        render.Padding(
+                            pad = 0,
+                            color = time_color,
+                            child = render.Animation([
+                                render.Row(
+                                    children = [
+                                        render.Image(DIGITS[char])
+                                        for char in now.format(time_format[0]).elems()
+                                    ],
+                                ),
+                                render.Row(
+                                    children = [
+                                        render.Image(DIGITS[char])
+                                        for char in now.format(time_format[1]).elems()
+                                    ],
+                                ) if blink_time else None,
+                            ]),
+                        ),
+                    ],
+                ),
+                render.Row(
+                    main_align = "center",
+                    expanded = True,
+                    children = [
+                        render.Padding(
+                            pad = 0,
+                            child = render.Text(
+                                content = now_date,
+                                font = "tom-thumb",
+                                color = date_color,
+                            ),
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    )
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Location(
+                id = P_LOCATION,
+                name = "Location",
+                desc = "Location for the display of date and time.",
+                icon = "place",
+            ),
+            schema.Text(
```

Comment:
These colors should probably be a dropdown, which sucks because we'd have to limit the number of colors as well and presumably that's why you did it this way 😅 . We can work on getting you a color picker but it's going to take some time on our end. I'm hesitant to publish this with a text input because I imagine it will be pretty confusing to someone trying to enter the colors

Reply:
Ok, lets leave this hanging until the color picker materializes

Reply:
I took out these configurations for now.